packages/cli/src/commands/start.ts

```diff
@@ -186,6 +195,9 @@ async function askQuestions({ template: defaultTemplate }: Pick<StartOptions, "t
     }
     context.services = services;
   }
+  else if (context.template === "react") {
```

Comment:
I think that empty array should be default value
what do you thinkg? @ryoppippi